docs/sources/configure-client/language-sdks/dotnet.md

```diff
@@ -9,24 +9,53 @@ aliases:
 
 # .NET
 
-## Supported platforms
+Our .NET Profiler is a powerful tool designed to enhance the performance analysis and optimization of .NET applications. It seamlessly integrates with Pyroscope, offering real-time insights into the resource usage and bottlenecks within your .NET codebase. This integration empowers developers to pinpoint inefficiencies, improve application speed, and ensure resource-efficient operation.
 
-| Spy Name     | Type         | Linux | macOS | Windows | Docker |
-|:------------:|:------------:|:-----:|:-----:|:-------:|:------:|
-| dotnetspy    | `embedded`   |   ✅  |       |         |        |
+### Supported Profiling Types
 
-## Supported .NET versions:
- - .NET 6.0
- - .NET 7.0
+The .NET Profiler supports the following profiling types:
 
-## How to add .NET profiling to your application
+* CPU
+* Wall time
+* Allocations
+* Lock contention
+* Exceptions
+
+### Compatibility
+
+Our .NET profiler works with the following .NET versions:
+* .NET 6
+* .NET 7
+* .NET 8
+
+
+## Before you Begin
+
+### Set Up a Pyroscope Server
+
+To capture and analyze profiling data, set up a Pyroscope server. This can be:
+
+* A **local server** for development, or
+* A **remote server** for production use.
+
+For installation instructions, see our [Get Started]({{< relref "../../get-started" >}}) guide.
+
+
+### Using Grafana Cloud Profiles
+
+Grafana Cloud Profiles is a hosted Pyroscope service. It provides a fully managed Pyroscope server, so you don't have to worry about installing and maintaining your own server. It also provides a hosted Grafana instance for visualizing your profiling data. For more information, see [Grafana Cloud Profiles](/products/cloud/profiles-for-continuous-profiling/).
+
+<!-- TODO: add a section like "Learn more about reading flamegraphs and using our product" once it's ready -->
```

Comment:
We have content for flamegraphs, see this PR: https://github.com/grafana/pyroscope/pull/2855

This content will be merged shortly, so you can put links to it if you want

Reply:
```suggestion
To learn more about using the Pyroscope UI and flamegraphs, refer to [Ingest and analyze profile data]({{< relref "../../../ingest-and-analyze-profile-data" >}}).
```